cmplibrary/src/test/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLibTest.java

```diff
@@ -103,23 +104,23 @@ public void run_followed_by_show_view(){
         lib.run();
```

Comment:
we should not call nor test any method on the setup. How did it get here? 😓

Reply:
@leodoin  This is not setup method it is run_followed_by_show_view() test method.